src/routes/tabs.js

```diff
@@ -24,26 +24,79 @@ const Tabs = () => {
           borderWidth: 1,
           borderTopWidth: 1,
         },
+        tabBarActiveTintColor: 'blue',
       }}>
       <TabNavigator.Screen
         name="Home"
         component={HomeScreen}
-        options={{headerShown: false}}
+        options={{
+          headerShown: false,
+          tabBarIcon: ({focused}) => {
+            return (
+              <View>
+                <Image
+                  source={require('../assets/home.png')}
+                  resizeMode="contain"
+                  style={{width: 25, tintColor: focused ? 'blue' : null}}
+                />
+              </View>
+            );
+          },
+        }}
       />
       <TabNavigator.Screen
         name="Store"
         component={StoreScreen}
-        options={{headerShown: false}}
+        options={{
+          headerShown: false,
+          tabBarIcon: ({focused}) => {
+            return (
+              <View>
+                <Image
+                  source={require('../assets/store.png')}
+                  resizeMode="contain"
+                  style={{width: 25, tintColor: focused ? 'blue' : null}}
```

Comment:
@Charan017 Please use `colors.js` or similar for all the color constants for reusability.

Reply:
Done